### PR TITLE
#31669 Updates for houdini engine release

### PR DESCRIPTION
**Pull request by josh-t**
* makes houdini available in same environments as maya by default
* shotgunpanel installed by default for houdini engine
* houdini executable defaults to houdini 15
* mac path added for houdini since it is now supported

### CODE REVIEW

File: env/asset_step.yml [hunk 1]
```diff
@@ -298,11 +298,23 @@ engines:
...
         template_work_area: asset_work_area_houdini
         version_compare_ignore_fields: []
+      tk-multi-shotgunpanel:
+        location:
+          version: v1.2.3
+          type: app_store
+          name: tk-multi-shotgunpanel
+        action_mappings:
+          Task:
+          - filters: {}
```
Comment (manneohrstrom) [inline env/asset_step.yml]: This is one of the cases when i feel the expanded yaml is less readable than the compact. In other places we have this format:

```
        action_mappings:
          Task:
          - actions: [assign_task, task_to_ip]
            filters: {}
        actions_hook: '{self}/tk-maya_actions.py'
```

(note also the inclusion of `task_to_ip`. I suggest changing houdini to use this form.

File: env/includes/app_launchers.yml [hunk 2]
```diff
@@ -87,7 +87,7 @@ launch_houdini:
     type: app_store
     name: tk-multi-launchapp
   mac_args: ''
-  mac_path: ''
+  mac_path: '@houdini_mac'
```
Comment (manneohrstrom) [inline env/includes/app_launchers.yml]: :smile:

**Comment by manneohrstrom** [comment]
looks sweet!

File: env/asset_step.yml (hunk 1 above)
Comment (josh-t) [inline env/asset_step.yml]: made consistent.

**Pull request closed without merging**